mlflow/spark.py

```diff
@@ -192,7 +193,18 @@ def save_model(spark_model, path, mlflow_model=Model(), conda_env=None, jars=Non
     mlflow_model.save(os.path.join(path, "MLmodel"))
 
 
-def load_model(path, run_id=None, dfs_tmpdir=None):
+def _load_model(model_path, dfs_tmpdir=None):
+    tmp_path = _tmp_path(dfs_tmpdir)
```

Comment:
This would crash if dfs_tmpdir is None, which the method seems like it would allow (given `dfs_tmpdir=None` as the default)